packages/web-components/src/popover/README.md

```diff
@@ -0,0 +1,93 @@
+| offset                              | Split into two DOM attributes offset-x and offset-y                             | Lets you displace a positioned element from its reference element.                                                                         |
+| overflowBoundary                    | Omitted - the boundary element is the trigger element                           |                                                                                                                                            |
+| overflowBoundaryPadding             | Omitted - not in design guidelines                                              | Applies a padding to the overflow boundary, so that overflow is detected earlier before the positioned surface hits the overflow boundary. |
+| pinned                              | Achieved through DOM Attribute                                                  | Disables automatic repositioning                                                                                                           |
+| position                            | Achieved through the positioning attribute or the popover-position attribute    | "above" \| "below" \| "before" \| "after"                                                                                                  |
+| strategy                            | Achieved through CSS                                                            | Specifies the type of CSS position property to use.                                                                                        |
+| useTransform                        | Omitted - not in design guidelines                                              | Modifies whether popover is positioned using transform.                                                                                    |
+| matchTargetSize                     | Omitted - not in design guidelines                                              | When set, the positioned element matches the chosen dimension(s) of the target element                                                     |
+
+\*[Positioning Shorthand](https://react.fluentui.dev/?path=/docs/concepts-developer-positioning-components--default#shorthand-positions)
+
+### type Placement (floating-ui) vs PositionShorthand (FluentUI)
+
+The popover web component uses floating-ui (a javascript npm package) to implement the design guidelines. FluentUI Design guidelines have two concepts in regard to positioning:
+
+1. Position - "above" | "below" | "before" | "after"
+2. Alignment - "start" | "end" | "center"
+
+Below is a maping of the floating ui Placement type to the FluentUI PositioningShorthand type. There are some differences in language, but the meanings in the ui are the same. For the sake of clarity the web component implementation precisely matches the language of the react implementation:
```

Comment:
Do you anticipate taking a dependency on FloatingUI for the positioning logic or do you plan to implement this on your own?

Reply:
We will be using the popover api and css anchor positioning if possible.